cypress/objects/timeLoginPage.js

```diff
@@ -0,0 +1,38 @@
+class TimeLoginPage {
+    visit() {
+        cy.visit('/')
+    }
+
+    getURL() {
+        return cy.url().should('include', '/time-logging')
```

Comment:
Right now this function not only gets the URL address, but it also asserts that it contains 'time-logging'. I'd probably drop the "should" part and add it in the test cases instead of this function's definition.